SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStrategySystem.cpp

```diff
@@ -178,6 +178,23 @@ void FSpatialStrategySystem::Flush(ISpatialOSWorker& Connection)
 	FMigrationContext Ctx(MigratingEntities, ModifiedEntities, DataStorages.EntitiesRemoved);
 	Strategy->CollectEntitiesToMigrate(Ctx);
 
+	for (const auto& StrategyMigration : Ctx.EntitiesToMigrate)
```

Comment:
Since the strategy isn't aware of ownership sets, we need to store its assignments separately.